migrations/20180408225001-create-device-name.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = {
+    up: function (queryInterface, Sequelize) {
+        return queryInterface.createTable('DeviceNames', {
+            id: {
+                allowNull: false,
+                autoIncrement: true,
+                primaryKey: true,
+                type: Sequelize.INTEGER
```

Comment:
All Sequelize.(type) need to be changed to DataTypes.(type) It gave me an error in other projects when trying to use Sequelize

Reply:
Disregard this